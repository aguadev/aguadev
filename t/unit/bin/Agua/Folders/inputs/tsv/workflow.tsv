testuser	Project1	Workflow1	1				
testuser	Project1	elandRef	2				
testuser	Project1	novoRef	3				[{"copiedby":"testuser","original":{"number":"3","project":"Project1","notes":"","name":"WorkflowA","description":"","username":"testuser"},"date":"2012-03-18 21:40:36"},{"original":{"number":"5","notes":"","project":"Project1","name":"WorkflowA","description":"","username":"testuser"},"copiedby":"testuser","date":"2012-03-18 21:40:36"}]
testuser	ProjectX	Workflow1	1				
testuser	ProjectX	Workflow2	2				
testuser	ProjectX	Workflow3	3				
testuser	ProjectY	Workflow1	1				
testuser	ProjectY	Workflow2	2				
