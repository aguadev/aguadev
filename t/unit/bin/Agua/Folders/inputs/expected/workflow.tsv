testuser	Project1	downloadFiles	1				[{"original":{"number":"1","status":"","project":"Project1","name":"Workflow1","description":"","username":"testuser","notes":""},"copiedby":"","date":null}]
testuser	Project1	elandRef	2				
testuser	Project1	novoRef	3				[{"copiedby":"testuser","original":{"number":"3","project":"Project1","notes":"","name":"WorkflowA","description":"","username":"testuser"},"date":"2012-03-18 21:40:36"},{"original":{"number":"5","notes":"","project":"Project1","name":"WorkflowA","description":"","username":"testuser"},"copiedby":"testuser","date":"2012-03-18 21:40:36"}]
